hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/ratis/TestOzoneManagerDoubleBuffer.java

```diff
@@ -18,9 +18,7 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
+import java.util.*;
```

Comment:
Please don't use * in import.

Reply:
Checkstyle agrees ;)

https://github.com/ArafatKhan2198/ozone/actions/runs/6920030792/job/18824061794#step:7:12